requestor/src/main/java/de/rwth/imi/flare/requestor/SearchQueryStringBuilder.java

```diff
@@ -79,6 +96,35 @@ private void constructQueryString(){
         appendTimeConstraints();
     }
 
+    private void appendSingleAgeComparison(Double age, Comparator comparator){
+        this.sb.append("birthdate=");
+        switch (comparator.toString()) {
+            case "gt" -> this.sb.append("lt");
+            case "lt" -> this.sb.append("gt");
+            case "ge" -> this.sb.append("le");
+            case "le" -> this.sb.append("ge");
+            case "eq" -> this.sb.append("eq");
```

Comment:
@bastianschaffer: You cannot handle "eq"/"ne" like this. For "eq" you want to evaluate if the birthdate depends on the unit within the range between today and today - 364 days for the unit year, today and today - 30 days for the unit month and so on. 
Ideally, considering the different lengths of a month and leap years.
Currently, we do not support "ne" from the UI. If you want to add it anyhow the same concept as here applies.